tests/test-stm-pin-parsing.stanza

```diff
@@ -0,0 +1,481 @@
+    "VDD3V3_USBHS"                 => "VDD3V3_USBHS",
+    "VDD50USB"                     => "VDD50USB",
+    "VDD50_USB"                    => "VDD50_USB",
+    "VDDA"                         => "VDDA",
+    "VDDA/VREF+"                   => "VDDA",
+    "VDDA1V1_REG"                  => "VDDA1V1_REG",
+    "VDDA1V8_DSI"                  => "VDDA1V8_DSI",
+    "VDDA1V8_REG"                  => "VDDA1V8_REG",
+    "VDDA1V8_Unused"               => "VDDA1V8_Unused",
+    "VDDCAP"                       => "VDDCAP",
+    "VDDCORE"                      => "VDDCORE",
+    "VDDDSI"                       => "VDDDSI",
+    "VDDIO2"                       => "VDDIO2",
+    "VDDLDO"                       => "VDDLDO",
+    "VDDMMC"                       => "VDDMMC",
+    "VDDPA"                        => "VDDPA",
+    "VDDPHYHS"                     => "VDDPHYHS",
+    "VDDQ_DDR"                     => "VDDQ_DDR",
+    "VDDRF"                        => "VDDRF",
+    "VDDRF1V55"                    => "VDDRF1V55",
+    "VDDSD"                        => "VDDSD",
+    "VDDSD12"                      => "VDDSD12",
+    "VDDSD3"                       => "VDDSD3",
+    "VDDSDMMC"                     => "VDDSDMMC",
+    "VDDSMPS"                      => "VDDSMPS",
+    "VDDUSB"                       => "VDDUSB",
+    "VDD_ANA"                      => "VDD_ANA",
+    "VDD_DSI"                      => "VDD_DSI",
+    "VDD_PLL"                      => "VDD_PLL",
+    "VDD_PLL2"                     => "VDD_PLL2",
+    "VDD_USB"                      => "VDD_USB",
+    "VDD_Unused"                   => "VDD_Unused",
+    "VFBSMPS"                      => "VFBSMPS",
+    "VLCD"                         => "VLCD",
+    "VLXSMPS"                      => "VLXSMPS",
+    "VREF+"                        => "VREF+",
+    "VREF-"                        => "VREF",
+    "VREFSD+"                      => "VREFSD+",
+    "VREFSD-"                      => "VREFSD",
+    "VREF_+"                       => "VREF_+",
+    "VR_PA"                        => "VR_PA",
+    "VSS EXPOSED PAD"              => "VSS",
+    "VSS"                          => "VSS",
+    "VSS/VSSA"                     => "VSS",
+    "VSSA"                         => "VSSA",
+    "VSSA/VREF-"                   => "VSSA",
+    "VSSDSI"                       => "VSSDSI",
+    "VSSRF"                        => "VSSRF",
+    "VSSSD"                        => "VSSSD",
+    "VSSSD/VREFSD-"                => "VSSSD",
+    "VSSSMPS"                      => "VSSSMPS",
+    "VSS_ANA"                      => "VSS_ANA",
+    "VSS_DSI"                      => "VSS_DSI",
+    "VSS_PLL"                      => "VSS_PLL",
+    "VSS_PLL2"                     => "VSS_PLL2",
+    "VSS_USBHS"                    => "VSS_USBHS"
+  ]
+
+  for [input, output] in kvs(pinnames) do :
+    var ret = extract-pin-name(input)
+    ret = to-string(ref-ify(ret))
+    #ASSERT(ret == output)
+
+defn kvs<?K,?V> (xs:Seqable<KeyValue<?K,?V>>) :
```

Comment:
I copied pasted `kvs` from lang-utils because I had isolated the test as pure stanza code. Not relevant now that we have jitx as dependency. It doesn't matter much that is just 2 lines